packages/generators/generators/lib/templates/core-controller.js.hbs

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+/**
+ *  {{ id }} controller
```

Comment:
Why is there both an ID & a UID there (since you only provide a UID in the generators)?